lib/spreewald_support/mail_finder.rb

```diff
@@ -58,14 +58,15 @@ def header_representation(mail)
     end
 
     def email_text_body(mail, type = '')
-      if mail.html_part && type != 'plain-text'
+      body = if mail.html_part && type != 'plain-text'
```

Comment:
Looks good to me - it gets my tests to run again, thanks!

Please add a CHANGELOG entry (nonbreaking) that explains that 3.0 re-introduced support for CLRF mails